packages/meditrak-server/src/kobo/startSyncWithKoBo.js

```diff
@@ -62,7 +61,7 @@ export async function syncWithKoBo(models, dataBroker, serviceCode) {
         }
         const entity = await transactingModels.entity.findOne({ code: responseData.orgUnit });
         if (!entity) {
-          winston.warn(`Skipping KoBo sync for unknown entity ${responseData.orgUnit}`);
+          await syncService.log(`Skipping KoBo sync for unknown entity ${responseData.orgUnit}`);
```

Comment:
Can we add more info here that might allow us to retry the record, e.g. the id?

Reply:
Done